src/pybind/cephfs/cephfs.pyx

```diff
@@ -1541,3 +1589,172 @@ cdef class LibCephFS(object):
             raise make_ex(ret, "error in lseek")
 
         return ret      
+
+    def utime(self, path, times=None):
+        """
+        Set access and modification time for path
+
+        :param path: file path for which timestamps have to be changed
+        :param times: if times is not None, it must be a tuple (atime, mtime)
+        """
+
+        self.require_state("mounted")
+        path = cstr(path, 'path')
+        if times:
+            if not isinstance(times, tuple):
+                raise TypeError('times must be a tuple')
+            if not isinstance(times[0], (int, float)):
+                raise TypeError('atime should be an int')
+            if not isinstance(times[1], (int, float)):
+                raise TypeError('mtime should be an int')
+        actime = modtime = int(time.time())
+        if times:
+            actime = int(times[0])
+            modtime = int(times[1])
```

Comment:
You wouldn't need this if you already check the types of times[0] and times[1]

Reply:
one could pass in a floating point value but utime() has seconds granularity -- so we'd need to typecast to int.

Reply:
note that the check above is `(int, float)`

Reply:
the other way would be to error out if float value is passed -- I think that would be the correct behaviour, but only for [f]utime() case (other call need to handle accepting int).